src/keys-builder/template/pipe.extractor.ts

```diff
@@ -37,32 +45,39 @@ interface KeyWithParam {
   paramsNode: AST;
 }
 
+function resolveKeyNode(ast: OrArray<AST>): LiteralPrimitive[] {
+  return coerceArray(ast)
+    .flatMap((expression) => {
+      if (isLiteralExpression(expression)) {
+        return expression;
+      } else if (isConditionalExpression(expression)) {
+        return resolveKeyNode([expression.trueExp, expression.falseExp]);
+      } else if (expression instanceof ParenthesizedExpression) {
+        return resolveKeyNode(expression.expression);
+      }
```

Comment:
If this is used for unwrapping the ParenthesizedExpression  expression, there is a utility function for it
https://github.com/angular/angular/blob/2e3925a65ce4280f8b31c4f8184666880d03acae/packages/core/schematics/utils/typescript/functions.ts#L28
We might be able to use it.